src/scrape_up/wuzzuf/wuzzuf.py

```diff
@@ -0,0 +1,125 @@
+import requests
+from bs4 import BeautifulSoup
+from time import sleep
+from stringdecorator import string_decorator
+
+
+class Jobs:
+    """
```

Comment:
I renamed the class, but I want you to rephrase the docstring, you can check modules like `GitHub` to see how docstrings are documented.